views/layouts/MainLayout/index.jsx

```diff
@@ -49,21 +50,23 @@ const MainLayout = () => {
   }, [])
 
   return (
-    <Box sx={{ display: 'flex', justifyContent: 'right' }}>
+    <div className="MainLayout_container">
```

Comment:
I'm confused why did we have to replace box with div. The class can be applied to the box component.

Reply:
it seemed unnecessary to use a Box when a simple div would do just fine. I'm wary of using these components unnecessarily because it can make upgrading down the road more of a pain that it needs to be.

Reply:
👍